codespell_lib/data/dictionary.txt

```diff
@@ -15019,6 +15034,7 @@ featurs->features
 feautre->feature
 feauture->feature
 feautures->features
+febbruary->february
```

Comment:
If you don't mind capitalised with `February`. Online dictionaries, browser, aspell, hunspell, ispell indicates a misspelling if it is in lower case. Thank you, if you don't mind adding these additional related 'months'.
`Februar` should be German, remove it if you think we should not add it.
```text
febbruary->February
februar->February

augest->August
novemeber->November
```
* https://github.com/search?q=febbruary&type=code
* https://github.com/search?q=februar&type=code
* https://github.com/search?q=augest&type=code
* https://github.com/search?q=novemeber&type=code